src/containers/contactAdmin/contactAdmin-list-container.js

```diff
@@ -64,6 +65,18 @@ class ContactAdminListWrapper extends Component {
   }
 }
 
+ContactAdminListWrapper.defaultProps = {
+  history: {},
+  getIssues: () => {},
+  contactAdminList: {},
```

Comment:
Maybe list is array?

Reply:
Nope. Look on 47 line, we did destructuring object, so it's object